src/irrlicht_modifications/static_text.cpp

```diff
@@ -0,0 +1,678 @@
+// Copyright (C) 2002-2012 Nikolaus Gebhardt
+// Copyright (C) 2016 Nathanaël Courant
```

Comment:
Is there any way you could write a comment detailing what modifications you actually made to this file?